packages/gestalt/src/RadioButton.css

```diff
@@ -1,6 +1,6 @@
 :root {
   --g-border-sm: 6px;
```

Comment:
what happens with out of the system px? `6px;`

Reply:
removed